File: ContainsDupe.java
class Solution {
    public static void main(String[] args)
    {
        int[] testCase = {1,2,3,1};

        System.out.println(containsDuplicate(testCase));
    }

    public static boolean containsDuplicate(int[] nums)
    {
        Boolean foundDupe = false;
        for(int i = 0; i < nums.length; i++)
        {
            for(int j = 0; j < nums.length; j++)
            {
               // System.out.println(nums[i]); // Debugging
                if(j != i && nums[i] == nums[j]) // If the numbers match & aren't from the same index
                    {
                        foundDupe = true;
                        break;
                    }
                /* else // Debugging
                    System.out.println("Not equal"); */
            }
            if(foundDupe)
                break;
        }
        return foundDupe;
    }
}

File: ContainsDupe.py
class Solution(object):
    def containsDuplicate(self, nums):
        localNums = nums
        hasDupe = False
        for i in range(len(nums)):
            for j in range(len(nums)):
                if(i != j & nums[i] == nums[j]):
                    hasDupe = True
                    break
            if(hasDupe):
                break
        return hasDupe

testcases = [1,2,3]
obj = Solution
contains = obj.containsDuplicate(obj, testcases)
print(contains)